llvm_util/utils.h

```diff
@@ -29,7 +30,7 @@ IR::Type& get_int_type(unsigned bits);
 IR::Type* llvm_type2alive(const llvm::Type *ty);
 
 IR::Value* make_intconst(uint64_t val, int bits);
-IR::Value* get_operand(llvm::Value *v);
+IR::Value* get_operand(llvm::Value *v, const llvm::DataLayout &DL);
```

Comment:
this makes all calls more verbose. it's better to pass this to init_llvm_utils() below.

Reply:
Yep, updated.